### PR TITLE
chore: streamline logging across modules 2

### PR DESCRIPTION
**Pull request by stijnmoreels**
Streamline logging and level across Powershell modules from `Arcus.Scripting.Management` to `Arcus.Scripting.Storage.Table`, alphabetically.

Relates to #155

### CODE REVIEW

File: src/Arcus.Scripting.Sql/Scripts/Invoke-AzSqlDatabaseMigration.ps1 [hunk 1]
```diff
@@ -48,22 +47,20 @@ function Get-SqlScriptFileText([string] $scriptPath, [string] $fileName) {
...
+                              "   [MinorVersionNumber] INT NOT NULL, " +
+                              "   [PatchVersionNumber] INT NOT NULL, " +
+                              "   [MigrationDescription] [nvarchar](256) NOT NULL, " +
                               "   [MigrationDate] DATETIME NOT NULL " +
-		                      "   CONSTRAINT [PK_DatabaseVersion] PRIMARY KEY CLUSTERED  ([MajorVersionNumber],[MinorVersionNumber],[PatchVersionNumber]) " +
-                              "				WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) " +
-	                          ") " +
+                              "   CONSTRAINT [PK_DatabaseVersion] PRIMARY KEY CLUSTERED  ([MajorVersionNumber],[MinorVersionNumber],[PatchVersionNumber]) " +
+                              "                WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) " +
+                              ") " +
```
Comment (stijnmoreels) [inline src/Arcus.Scripting.Sql/Scripts/Invoke-AzSqlDatabaseMigration.ps1:63]: Changed the tab characters to spaces.

File: src/Arcus.Scripting.Management/Scripts/Get-AzApiManagementSoftDeletedResources.ps1 [hunk 2]
```diff
@@ -6,15 +6,15 @@ param(
     [Parameter(Mandatory = $true)][string] $ApiVersion
 )
 
-Write-Verbose "Checking if the API Management instance with name '$Name' is listed as a soft deleted service"
+Write-Verbose "Checking if the zure API Management service '$Name' is listed as a soft deleted service..."
```
Comment (pim-simons) [inline src/Arcus.Scripting.Management/Scripts/Get-AzApiManagementSoftDeletedResources.ps1:9]:
```suggestion
Write-Verbose "Checking if the Azure API Management service '$Name' is listed as a soft deleted service..."
```

File: src/Arcus.Scripting.Management/Scripts/Restore-AzApiManagementSoftDeletedService.ps1 [hunk 3]
```diff
@@ -36,7 +36,7 @@ try {
     $putUri = "$resourceManagerUrl" + "$serviceId" + "?api-version=$ApiVersion"
     $restoreService = Invoke-RestMethod -Method PUT -Uri $putUri -ContentType 'application/json' -Headers $authHeader -Body $body
 } catch {
-    throw "The soft deleted API Management instance '$Name' could not be restored. Details: $($_.Exception.Message)"
+    throw "Soft deleted Azure API Management serivce '$Name' could not be restored. Details: $($_.Exception.Message)"
```
Comment (pim-simons) [inline src/Arcus.Scripting.Management/Scripts/Restore-AzApiManagementSoftDeletedService.ps1:39]:
```suggestion
    throw "Soft deleted Azure API Management service '$Name' could not be restored. Details: $($_.Exception.Message)"
```

File: src/Arcus.Scripting.Management/Scripts/Restore-AzApiManagementSoftDeletedService.ps1 [hunk 4]
```diff
@@ -36,7 +36,7 @@ try {
...
 }
 
-Write-Host "Successfully restored the soft deleted API Management instance '$Name'"
\ No newline at end of file
+Write-Host "Successfully restored the soft deleted API Management service '$Name'"
```
Comment (pim-simons) [inline src/Arcus.Scripting.Management/Scripts/Restore-AzApiManagementSoftDeletedService.ps1:42]:
```suggestion
Write-Host "Successfully restored the soft deleted API Management service '$Name'" -ForegroundColor Green
```

**Review by pim-simons — CHANGES_REQUESTED** [review]
Great work again 👍🏻 
[As mentioned in the other PR](https://github.com/arcus-azure/arcus.scripting/pull/351#pullrequestreview-1229987141) we should probably also update the docs.

**Review by pim-simons — APPROVED** [review]
Looks good to me 🥇

**Pull request merged**